### ISSUE TITLE
Leskovets Issues

### ISSUE THREAD

**Issue by Nels0** [scooby78/160thSOR#86]
AI have issues following roads and moving between different roads.

Witnessed by Barry and Nels0

**Comment by Nels0** [comment]
Render distance issues when flying in a jet: (BuckKaneer) 
 - Roads disappear
 - Towns disappear
 - Ground vehicles disappear into the ground

**Comment by Diesel486** [comment]
reg render distance. I've seen this before on other maps, is it possible this is something within the maps config to reduce framerate issues? alongside that, should we be worried about it? considering certain people complain that they can see everything on the map in air? (Buck)

**Comment by scooby78** [comment]
@Diesel yeah I don’t think the rendering is going to be a major issue to the hole community but the path finding is of more concern to me..

**Comment by Nels0** [comment]
I understand it's only for wheeled/tracked vehicles. Likely can be worked around in the short term, and maybe we just rotate out the map in the next few months.

**Comment by scooby78** [comment]
@Nels0 yes that’s correct it will most likely not remaining in the pack for any extended period.
